src/Compilers/Core/Portable/Symbols/Attributes/ObsoleteAttributeData.cs

```diff
@@ -1,43 +1,44 @@
 ﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
 
 using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
-using Microsoft.CodeAnalysis.Text;
 
 namespace Microsoft.CodeAnalysis
 {
+    internal enum ObsoleteAttributeKind
+    {
+        None,
```

Comment:
`None` is intended for use by the `Uninitialized` instance only, although other callers are not prevented from using it.